main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+spring.datasource.url=jdbc:postgresql://localhost/people_ms
+spring.datasource.username=postgres
+spring.datasource.password=sAU-uf,S5W=^/)Yu
```

Comment:
You should never ever add actual password to properties. Just a thought, it's okay here because this is an example but in real world applications, this must always be encrypted.

Reply:
Thanks Gabz, I changed this by encrypting the password with Jasypt and storing it in a file called 'encrypted.properties' which I won't be sharing on GitHub.